docker/Dockerfile

```diff
@@ -6,8 +6,6 @@ ARG NUM_PROCESSES
 ENV NUM_PROCESSES ${NUM_PROCESSES:-4}
 ARG NUM_THREADS
 ENV NUM_THREADS ${NUM_THREADS:-2}
-ARG WSGI_CONTAINER
```

Comment:
Is NUM_THREADS used anymore? 

Also we should remove WSGI_CONTAINER (and NUM_THREADS if the answer to above is no) from the Helm chart `values.yaml` file, under `env`.

Reply:
No It is not needed as Num threads are Required only when WSGI_CONTAINER is uwsgi, since we will always use Gunicorn , it should be ok to remove Num Threads from Dockerfile
Also removed the WSGI_CONTAINER  from values.yaml and also removed `uwsgi` from the file setup.py as it is being removed, tested these new code changes and the driver works fine and is handling graceful shutdown

![image](https://user-images.githubusercontent.com/79189065/135212709-4207e968-ca79-4862-8d25-e85d48e8e790.png)